README.md

```diff
@@ -89,7 +89,7 @@ Some of the most common built-in matchers are
 - `s.object(fields)`
 - `s.objectWithOnly(fields)`
 - `s.regex(reg)`
-- `s.string()`
+- `s.string(min, max)`
```

Comment:
Whatever we do, we should be consistent, for example changing `s.number(min, max)` as well.